udf/array_drop_first_and_append.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TEMP FUNCTION
+  udf_array_drop_first_and_append(arr ANY TYPE, append ANY TYPE) AS (
+    ARRAY_CONCAT(
+      ARRAY(
+        SELECT v
+        FROM UNNEST(arr) AS v WITH OFFSET off
+        WHERE off > 0
+        ORDER BY off ASC),
```

Comment:
this order by clause shouldn't have any effect on the result

Reply:
It should retain the original ordering of the array, sans the first element: (see note [here](https://cloud.google.com/bigquery/docs/reference/standard-sql/arrays#filtering-arrays))